src/dom/rect.js

```diff
@@ -234,21 +247,63 @@ export default class Rect {
 	}
 
 	/**
+	 * Checks if all properties ({@link #top}, {@link #left}, {@link #right},
+	 * {@link #bottom}, {@link #width} and {@link #height}) are the same as in the other `Rect`.
```

Comment:
are the same -> equal? properties are "the same"  has a bit different meaning than "values of these properties equal ...". It's a detail and shouldn't be confusing, but I found the wording a bit unusual.